.github/workflows/flutter_ios.yaml

```diff
@@ -0,0 +1,68 @@
+name: Flutter iOS
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        description: The working directory relative to the repo root
+        required: true
+        type: string
+      package-name:
+        description: The name of the app-facing (or federated) iOS package being tested
+        required: true
+        type: string
+
+# Cancels in-progress job when there is another push to same ref.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Test
+    runs-on: macos-latest
+    timeout-minutes: 60
+    strategy:
+      # Allows other matrix items to run if one fails
+      fail-fast: false
+      matrix:
+        sdk:
+          - stable
+          - beta
```

Comment:
Instead of testing different Flutter versions (or maybe in addition to) can we test different Xcode or iOS versions? 

It looks like `macos-latest` (which is `macos-12`) uses Xcode 14.2 and `macos-11` uses Xcode 13.2. But it might be worth to compile for iOS 13 and iOS 16 - our min + current stable

Doesn't have to be this PR, just food for thought

Reply:
I agree, we should add iOS version matrix. I've been thinking about doing this in integration tests, too, maybe only running them on a smaller schedule load than other tests and adding extra matrices, as well. Will think about more for a future PR.

Reply:
👍 I think the regular tests should always run on the minimum version that we support. Others can run with longer interval. Beta version of Xcode is not offered correct?

Reply:
I don't think beta is guaranteed to be available, the versions, sims and CLI tools they have are on https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md